code/__DEFINES/clothing.dm

```diff
@@ -122,6 +122,20 @@
 #define FEMALE_UNIFORM_FULL			1
 #define FEMALE_UNIFORM_TOP			2
 
+//flags for alternate styles: These are hard sprited so don't set this if you didn't put the effort it

```

Comment:
# ACCORDING TO ALL KNOWN LAWS OF AVIATION

# THIS PR SHOULD NOT BE ABLE TO FLY